FirebaseFunctionsSwift/Sources/Functions.swift

```diff
@@ -126,8 +140,10 @@ private enum Constants {
     Response: Decodable>(_ name: String,
                          requestAs: Request.Type = Request.self,
                          responseAs: Response.Type = Response.self,
-                         encoder: FirebaseDataEncoder = FirebaseDataEncoder(),
-                         decoder: FirebaseDataDecoder = FirebaseDataDecoder())
+                         encoder: FirebaseDataEncoder = FirebaseDataEncoder(
+                         ),
```

Comment:
Yikes, this formatting is a big rough. Curious if there's a way we can fix this later in swift lint

Reply:
Maybe there's an option or we should file a bug - but prefer to defer this now - until a later polish phase.